paper/paper.md

```diff
@@ -89,7 +89,7 @@ Python scripting, or even manual correction in spreadsheet software.
 
 With TAXPASTA, all of those formats can be converted into a single, standardised
 output, that, at a minimum, contains taxonomic identifiers and their relative
-abundances as integer counts. It can also be used to aggregate profiles across
+sequence abundances as integer counts. It can also be used to aggregate profiles across
```

Comment:
It's not only sequence abundance, so I would not write that here.

Reply:
Ah ok, I thought it was!